### DISCUSSION TITLE
Unable to show data when graph is inside a plain old fragment

### DISCUSSION THREAD

**Discussion by git-vcs** [patrykandpatrick/vico#957] [Questions] [answered]
### Question

Hello firstly I would just like to say that this library looks awesome.
I am a bit unsure if this is a bug or I have misunderstood  the documentation.
I made a new project based on Androids (`Basic view activity` boilerplate) and proceeded with the `getting-started` guide for `version 2.0.0-beta3`.

 I added this to `fragment_first.xml` :
```
    	  	<com.patrykandpatrick.vico.views.cartesian.CartesianChartView
        	app:layout_constraintStart_toStartOf="parent"
        	app:layout_constraintTop_toTopOf="parent"
        	app:layout_constraintEnd_toEndOf="parent"
        	android:id="@+id/cartesian_chart_view"
        	android:layout_width="match_parent"
        	android:layout_height="wrap_content"
        	app:layers="column"
        	app:showBottomAxis="true"
        	app:showStartAxis="true" />
```

I then added this to `FirstFragment.kt`:

```
	override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
    	super.onViewCreated(view, savedInstanceState)
    	val modelProducer = CartesianChartModelProducer()
    	with(binding){
        	cartesianChartView.modelProducer = modelProducer
        	lifecycleScope.launch {
            	modelProducer.runTransaction { columnSeries { series(4, 12, 8, 16) } }
        	}
    	}

    	binding.buttonFirst.setOnClickListener {
        	findNavController().navigate(R.id.action_FirstFragment_to_SecondFragment)
    	}
	}
```

 Code compiles and runs and I can see in the layout inspector that the view is there. But it’s completely white and there is nothing inside it.

I tried following the guide for version `vico = "1.15.0" ` and it worked just fine.

The example code(from the repository) works on my machine. But I am not that familiar with jetpack compose.

To sum up I am trying to add a graph inside an android project that does not have jetpack compose.

### Vico version(s)

2.0.0-beta3

### UI framework(s)

Views

**Comment by Gowsky** [comment, marked as answer]
Hello, @git-vcs. It looks like the view starter examples were outdated. Vico `2.0.0-beta.3` introduces some changes to XML styling. We've updated the examples — please check the guide again and update your code accordingly. The chart should then work properly.

Also keep in mind that, as per the guide, `val modelProducer = CartesianChartModelProducer()` should be e.g. at the `Fragment` level, not in `onViewCreated`.
